lib/v2/jsu/radar.js

```diff
@@ -0,0 +1,35 @@
+module.exports = {
+    'jsu.edu.cn': {
+        _name: '吉首大学',
+        cxzx: [
+            {
+                title: '吉首大学创新中心',
```

Comment:
No need to enter `_name` again in `title`
docs: https://docs.rsshub.app/joinus/new-radar#title
```suggestion
                title: '创新中心',
```